src/deploy/functions/services/firebaseAlerts.ts

```diff
@@ -0,0 +1,49 @@
+import * as backend from "../backend";
+import * as iam from "../../../gcp/iam";
+import { getProjectNumber } from "../../../getProjectNumber";
+import { FirebaseError } from "../../../error";
+
+export const SERVICE_ACCOUNT_TOKEN_CREATOR_ROLE = "roles/iam.serviceAccountTokenCreator";
+
+/**
+ * Finds the required project level IAM bindings for the Pub/Sub service agent
+ * If the user enabled Pub/Sub on or before April 8, 2021, then we must enable the token creator role
+ * @param projectId project identifier
+ * @param existingPolicy the project level IAM policy
+ */
+export function obtainFirebaseAlertsBindings(
```

Comment:
Is this really a Firebase Alerts feature or something that all event providers are going to need?

Reply:
NVM. See below comments. I think we could have had something like:

```
export function obtainFirebaseAlertBindings(projectNumber) {
  iam.ensureBinding(firebaseAlertRole, firebaseAlertPermission)
}
```

Reply:
You're correct, this is not a Firebase Alerts specific binding. It was yanked in #4324 and I added a noop instead